python/hidet/backend/codegen.py

```diff
@@ -715,11 +715,13 @@ def visit_Function(self, func: Function) -> Doc:
             block_dim = func.attrs['cuda.block_dim']
             if isinstance(block_dim, list):
                 block_dim = prod(block_dim)
-            if 'cuda.min_blocks' in func.attrs:
-                min_blocks = func.attrs['cuda.min_blocks']
-                doc += f' __launch_bounds__({block_dim}, {min_blocks})'
-            else:
-                doc += f' __launch_bounds__({block_dim})'
+            if isinstance(block_dim, (Constant, int)):
+                if 'cuda.min_blocks' in func.attrs:
+                    min_blocks = func.attrs['cuda.min_blocks']
+                    if isinstance(min_blocks, (Constant, int)):
+                        doc += f' __launch_bounds__({block_dim}, {min_blocks})'
+                else:
+                    doc += f' __launch_bounds__({block_dim})'
```

Comment:
When block_dim is constant, 'cuda.min_blocks' in func.attrs but 'cuda_min_blocks' is not constant, we should also set the launch bound for block_dim.

Reply:
```suggestion
            if isinstance(block_dim, (Constant, int)):
                if 'cuda.min_blocks' in func.attrs:
                    min_blocks = func.attrs['cuda.min_blocks']
                    if isinstance(min_blocks, (Constant, int)):
                        doc += f' __launch_bounds__({block_dim}, {min_blocks})'
                    else:
                        doc += f' __launch_bounds__({block_dim})'
                else:
                    doc += f' __launch_bounds__({block_dim})'
```